From fad0699c25b864e1ce2466c3c683007af5c350ad Mon Sep 17 00:00:00 2001
From: Ivana Hrivnacova <Ivana.Hrivnacova@cern.ch>
Date: Wed, 9 Mar 2022 18:03:18 +0100
Subject: [PATCH] Fix compilation with Geant4 11.0.p1: (#30)

Added missing test for G4VERSION_NUMBER in TG4RunManager.cxx
---
 source/run/src/TG4RunManager.cxx | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/source/run/src/TG4RunManager.cxx b/source/run/src/TG4RunManager.cxx
index 64a45f0e0..5190172b0 100644
--- a/source/run/src/TG4RunManager.cxx
+++ b/source/run/src/TG4RunManager.cxx
@@ -47,6 +47,7 @@
 #include <G4UIExecutive.hh>
 #include <G4UImanager.hh>
 #include <G4UIsession.hh>
+#include <G4Version.hh>
 #include <Randomize.hh>
 
 #ifdef USE_G4ROOT
@@ -405,8 +406,10 @@ void TG4RunManager::Initialize()
   // create G4RunManager
   // ConfigureRunManager();
 
+#if G4VERSION_NUMBER == 1100
   // Temporary work-around for bug in Cerenkov in Geant4 11.0
   TG4PhysicsManager::Instance()->StoreCerenkovMaxBetaChangeValue();
+#endif
 
   // initialize Geant4
   fRunManager->Initialize();
